atc/api/accessor/accessor.go

```diff
@@ -125,37 +127,37 @@ var requiredRoles = map[string]string{
 	atc.ListBuilds:                    "viewer",
 	atc.BuildEvents:                   "viewer",
 	atc.BuildResources:                "viewer",
-	atc.AbortBuild:                    "member",
+	atc.AbortBuild:                    "pipeline-operator",
 	atc.GetBuildPreparation:           "viewer",
 	atc.GetJob:                        "viewer",
-	atc.CreateJobBuild:                "member",
+	atc.CreateJobBuild:                "pipeline-operator",
 	atc.ListAllJobs:                   "viewer",
 	atc.ListJobs:                      "viewer",
 	atc.ListJobBuilds:                 "viewer",
 	atc.ListJobInputs:                 "viewer",
 	atc.GetJobBuild:                   "viewer",
-	atc.PauseJob:                      "member",
-	atc.UnpauseJob:                    "member",
+	atc.PauseJob:                      "pipeline-operator",
+	atc.UnpauseJob:                    "pipeline-operator",
 	atc.GetVersionsDB:                 "viewer",
 	atc.JobBadge:                      "viewer",
 	atc.MainJobBadge:                  "viewer",
-	atc.ClearTaskCache:                "member",
+	atc.ClearTaskCache:                "pipeline-operator",
 	atc.ListAllResources:              "viewer",
 	atc.ListResources:                 "viewer",
 	atc.ListResourceTypes:             "viewer",
 	atc.GetResource:                   "viewer",
-	atc.PauseResource:                 "member",
-	atc.UnpauseResource:               "member",
+	atc.PauseResource:                 "pipeline-operator",
+	atc.UnpauseResource:               "pipeline-operator",
 	atc.UnpinResource:                 "member",
 	atc.SetPinCommentOnResource:       "member",
```

Comment:
These both (`atc.UnpinResource`, `atc.SetPinCommentOnResource`) should probably be `pipeline-operator` too, since `atc.PinResourceVersion` lets them pin it.